app/javascript/react/components/BrandShowContainer.js

```diff
@@ -1,40 +1,37 @@
 import React, { useState, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import BrandShowTile from './BrandShowTile.js'
 
 const BrandShowContainer = (props) => {
-    const [pizzas, setPizzas] = useState([])
-    const [brand, setBrand] = useState({name: ''})
+  const [pizzas, setPizzas] = useState([])
+  const [brand, setBrand] = useState({name: ''})
 
-    let brandId = props.match.params.id
+  let brandId = props.match.params.id
```

Comment:
this should be `const`. Always prefer constant to let unless the value may be re-assigned